process/common.go

```diff
@@ -1009,3 +1009,38 @@ func CheckIfIndexesAreOutOfBound(
 
 	return nil
 }
+
+func SetBaseExecutionResult(ert ExecutionResultsTracker, blockChain data.ChainHandler) error {
+	if check.IfNil(blockChain) {
+		return ErrNilBlockChain
+	}
+	if check.IfNil(ert) {
+		return ErrNilExecutionResultsTracker
+	}
+
+	currentBlock := blockChain.GetCurrentBlockHeader()
+	if currentBlock == nil || !currentBlock.IsHeaderV3() {
+		return nil
```

Comment:
does this needs to be correlated with the round activation flag? the first header in async round activation will be with v3? or based on epoch activation?